### PR TITLE
Expose default settle and reveal timeouts in webui's config*.json

### PR DESCRIPTION
**Pull request by andrevmatos**

### CODE REVIEW

File: raiden/api/rest.py [hunk 1]
```diff
@@ -384,15 +384,29 @@ def _is_raiden_running(self):
 
     def _serve_webui(self, file_name='index.html'):  # pylint: disable=redefined-builtin
         try:
-            assert file_name
+            if not file_name:
```
Comment (LefterisJP) [inline raiden/api/rest.py]: Why the change? Just to raise `NotFound` instead of assertion exception?

Comment (andrevmatos) [reply raiden/api/rest.py]: From what I understand now, `AssertionError` should be used exclusively for errors which aren't expected/should be fatal. This change was just to replace _expected_ `assert` by one of the already handled errors (which is also coherent with the logic here)
